docs/features.md

```diff
@@ -0,0 +1,97 @@
+# Language
+
+* Ceremony master - TBD
+
+# Mainnet Feature Requirements
+
+## Network Launch
+
+### UC: As a Coop SRE I want to launch a network
+
+#### AC: A succesful genesis ceremony 
+
+##### test: [todo: requires integration test]
+##### steps:
+
+* `nodeA`(ceremony-master) is instatantied with flags `--required-sigs 2 --duration 5min --interval 10sec --bonds-file <holds two nodes nodeB and nodeC`.
+* `nodeB`(validator) and `nodeC`(validator) joins p2p, both poining `nodeA` as bootstrap
+* `nodeA` sends `UnapprovedBlock` to `nodeB` and `nodeC`
+* `nodeB` and `nodeC` receives `UnapprovedBlock`
+* `nodeB` and `nodeC` send back `BlockApproval`
+* `nodeA` transitions to `ApprovedBlockReceivedHandler`
+* `nodeA` sends `ApprovedBlock` to `nodeB` and `nodeC`
+* `nodeB` and `nodeC` transition to ApprovedBlockReceivedHandler
+* `show-blocks` on `nodeA` shows single block (genesis) where Bonds holds `nodeB` and `nodeC`
+* `show-blocks` on `nodeB` shows single block (genesis) where Bonds holds `nodeB` and `nodeC`
+* `show-blocks` on `nodeC` shows single block (genesis) where Bonds holds `nodeB` and `nodeC`
+
+#### AC: A succesful genesis ceremony with read-only nodes joining 
+
+##### test: [todo: requires integration test]
+##### steps:
+
+* `nodeA`(ceremony-master) is instatantied with flags `--required-sigs 2 --duration 5min --interval 10sec --bonds-file <holds two nodes nodeB and nodeC`.
+* `nodeB`(validator) and `nodeC`(validator) joins p2p, both poining `nodeA` as bootstrap
+* `nodeD`(read-only) joins p2p
+* `nodeA` sends `UnapprovedBlock` to `nodeB` and `nodeC`
+* `nodeB` and `nodeC` receives `UnapprovedBlock`
+* `nodeB` and `nodeC` send back `BlockApproval`
+* `nodeA` transitions to `ApprovedBlockReceivedHandler`
+* `nodeA` sends `ApprovedBlock` to `nodeB` and `nodeC`
+* `nodeB` and `nodeC` transition to ApprovedBlockReceivedHandler
+* `show-blocks` on `nodeA` shows single block (genesis) where Bonds holds `nodeB` and `nodeC`
+* `show-blocks` on `nodeB` shows single block (genesis) where Bonds holds `nodeB` and `nodeC`
+* `show-blocks` on `nodeC` shows single block (genesis) where Bonds holds `nodeB` and `nodeC`
+* `nodeD` **never** transitions to `ApprovedBlockReceivedHandler`
```

Comment:
This may be what happens now but eventually we want `nodeD` to continuously send any of the validators an ApprovedBlockRequest and receive the ApprovedBlock too once the ceremony ends. It just doesn't participate in the ceremony.

Reply:
Kent's right. Today nodeD will make the transition after the ceremony completes, when nodeA sends out the first approved block and all connected nodes transition. The thing to look for with nodeD is that may send a response to approve the UnapprovedBlock and nodeA will recognize it is not one of the validators.

Reply:
Ok, I will modify test accordingly